src/server/redis.js

```diff
@@ -6,7 +6,7 @@ import redisMock from 'redis-mock'
 const client =
   process.env.NODE_ENV === 'test'
     ? redisMock.createClient()
-    : redis.createClient(process.env.REDIS_HOST || '127.0.0.1')
+    : redis.createClient(6379, process.env.REDIS_HOST || '127.0.0.1')
```

Comment:
Are you sure you can set the port first? I fixed it the other way around in my other PR 😄

Reply:
From the doc https://github.com/NodeRedis/node_redis#rediscreateclient : 

```
redis.createClient(port[, host][, options])
```

Reply:
I think this signature is nicer:
```
redis.createClient([options])
```

Property | Default | Description
-- | -- | --
host | 127.0.0.1 | IP address of the Redis server
port | 6379 | Port of the Redis server

Reply:
Nevermind, we’re getting rid of Redis anyway!